core/store/migrate/migrations/0195_add_not_null_to_evm_chain_id_in_job_specs.go

```diff
@@ -3,16 +3,21 @@ package migrations
 import (
 	"context"
 	"database/sql"
+	"os"
 
 	"github.com/pkg/errors"
 	"github.com/pressly/goose/v3"
 )
 
+// No need to scan migration script
+// START-NOSCAN
 func init() {
 	goose.AddMigrationContext(Up195, Down195)
 }
 
 const (
+	EVMChainIDNotNullMigrationHelper = "EVM_CHAINID_MIGRATION_HELPER_ENV_VAR"
```

Comment:
Let's define this alongside the other environment variables in `package env`, and prefix it with `CL_`. I wonder if the name should have the migration number? :thinking: